client/src/components/questions/Questions.jsx

```diff
@@ -5,9 +5,26 @@ export default function Questions({
   feature
 }) {
 
+  const [questions, setQuestions] = useState([]);
+
+  useEffect(() => {
+    fetcher.questions.getById(feature.id)
+      .then(results => setQuestions(results.data.results))
+      .catch(err => console.log('err Questions on feature change fetch: ', err));
+  }, [feature]);
+
   return (
     <div>
       Questions and Answers
+      <div>
```

Comment:
Maybe move the mapper into an external function outside of the return() block? Stylistic preference